### ISSUE TITLE
Incorrect dependencies

### ISSUE THREAD

**Issue by DarkRedman** [EhsanKia/MumbleChatBot#1]
You claim that Twisted is the only dependency but no, it also needs python-protobuf which also use the google dependency, plus the code is old (python 2) and not compatible with the current version of python.

**Comment by EhsanKia** [comment]
Good catch, and yes, it's an old project of mine from years ago, but I believe Twisted is still not fully ported to Python 3.
